src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -161,6 +170,7 @@ export async function exportSavedObjectsToStream({
     savedObjectsClient,
     exportSizeLimit,
     namespace,
+    workspaces,
```

Comment:
since bulkGet don't consume workspaces, we could remove it for export

Reply:
The workspaces is need here as it may be used in find method.